server/server.js

```diff
@@ -13,11 +13,15 @@ app.use(express.static('../public/dist'))
 app.use(bodyParser.urlencoded({extended:false}));
 app.use(bodyParser.json());
 
-app.get('/menus', (req, res) => {
+app.get('/api/menus', (req, res) => {
   console.log('I am in the get', req.body);
-  //query the data base for the dog by id
+
   res.send(menuData);
 });
 
```

Comment:
I noticed that you have the restaurant id json included, but I don't see where the ids or names from the json are referenced.

Reply:
Yeah, I finally added them, I am glad your writing notes Alvin, definitely appreciate it!! They should be required at the top of the photosSeeder.js file

Reply:
Nice, man. Make sure you close the pull request after you've fixed everything too, though.

Reply:
man, I know there is a bunch of github steps that I am completely messing up on, thanks for the heads up Alvin

Reply:
Hey Alvin, I'm sorry to have to ask but I gotta. I've never closed a pull request, does that mean merge the pull request? How do I do that?

Reply:
Whoops, yeah just meant merge. Isn't there an option to merge on this page?